src/store.ts

```diff
@@ -1,24 +1,39 @@
 import raygun from "raygun4js";
-import { configureStore, Action, getDefaultMiddleware } from "@reduxjs/toolkit";
+import { configureStore, Action, getDefaultMiddleware, Middleware, Dispatch } from "@reduxjs/toolkit";
 import rootReducer, { RootState } from "./rootReducer";
 import { ThunkAction } from "redux-thunk";
 import {
   mixerMiddleware,
   mixerKeyboardShortcutsMiddleware,
   startNewsTimer,
 } from "./mixer/state";
-import { persistStore } from "redux-persist";
+import {
+  persistStore,
+  FLUSH,
+  REHYDRATE,
+  PAUSE,
+  PERSIST,
+  PURGE,
+  REGISTER
+} from "redux-persist";
+
+const raygunMiddleware: Middleware < {}, RootState, Dispatch < any >> = (store) => (next) => (action) => {
```

Comment:
I'm... not actually sure how this fixes anything? Looks identical to the old code, modulo type declarations

Reply:
Me neither, but it stopped breaking it / complaining haha. I'm now seeing raygun.js appearing in the console again too.

Reply:
Have you checked if it properly reports redux actions?